cfme/tests/cloud/test_provisioning.py

```diff
@@ -125,35 +125,47 @@ def vm_name(request):
     return random_vm_name('prov')
 
 
+@pytest.fixture(scope="function")
+def provision_check(provider):
```

Comment:
Why do you need this fixture? Why not to  create a regular function `provision_check(provider, instance, inst_args, image)`?

Reply:
don't have any good reason to use fixture here, so will convert it to function

Reply:
Done